device/bpi-m1.conf

```diff
@@ -1,11 +1,12 @@
-# http://www.banana-pi.org/
+# http://www.banana-pi.org/m1.html
+# http://www.banana-pi.org/m1plus.html
```

Comment:
One question.. I have the R1 and isn't it the same as M1 from boot perspective?

Reply:
from SMP kernel it's same because they use allwinner kernel conf. But I'm not sure from device tree blob (dtb) copied from u-boot. but report from @mimugmail he said that current bpi image won't boot. maybe current fix for allwinner I add on arm.sh will do the trick?

Reply:
let me take care of this... @mimugmail gave me a m1 and r1 for that particular reason a while back :)

Reply:
nice. hope that this image will boot. R1 looks like a nice board, got many wan/lan options.

Reply:
maybe some day r64 works, that one looks very nice.. it has PoE as well

Reply:
i'll take a look at it asap. but why all these boards only use 1gb ram? it got many potentials if have at least 2gb. maybe can hack the hardware to upgrade it's ram?